src/tx/builder/schema.js

```diff
@@ -752,8 +752,8 @@ const CHANNEL_FORCE_PROGRESS_TX = [
   TX_FIELD('fromId', FIELD_TYPES.id, 'ak'),
   TX_FIELD('payload', FIELD_TYPES.binary, 'tx'),
   TX_FIELD('round', FIELD_TYPES.int),
-  TX_FIELD('update', FIELD_TYPES.binary),
-  TX_FIELD('stateHash', FIELD_TYPES.binary),
+  TX_FIELD('update', FIELD_TYPES.binary, 'cb'),
```

Comment:
Actually, I'm not sure about this one, can't find a clear definition in both:
https://github.com/aeternity/protocol/blob/master/serializations.md#channel-solo-force-progress-transaction and
https://github.com/aeternity/protocol/blob/master/node/api/api_encoding.md#encoding-scheme-for-api-identifiers-and-byte-arrays